src/management/JobsManager.js

```diff
@@ -173,6 +173,7 @@ JobsManager.prototype.importUsers = function(data, cb) {
   var promise = options.tokenProvider.getAccessToken().then(function(access_token) {
     return axios
       .post(url, form, { headers: { ...headers, Authorization: `Bearer ${access_token}` } })
+      .then(({ data }) => data)
```

Comment:
this is probably a leftover from the other PR, the data extraction/transformation should happen in the introduced method and the old one should remain the same.